arithmetization/src/test/java/net/consensys/linea/zktracer/ZkTracerTest.java

```diff
@@ -30,4 +34,33 @@ public void createNewTracer() {
     final ZkTracer zkTracer = new ZkTracer(chainConfig);
     assertThat(zkTracer.isExtendedTracing()).isTrue();
   }
+
+  @Test
+  void tracedModuleForFork() {
+    final ZkTracer zkTracer = new ZkTracer(chainConfig);
+    final int allModule =
+        new ArrayList<>(
+                Stream.concat(
+                        zkTracer.getHub().realModule().stream(),
+                        zkTracer.getHub().refTableModules().stream())
+                    .toList())
+            .size();
+    final int tracedModule = zkTracer.getHub().getModulesToTrace().size();
+
+    switch (fork) {
+      case LONDON, PARIS, SHANGHAI -> {
+        checkArgument(
+            allModule == tracedModule + 2,
```

Comment:
`allModule` should be e.g. `totalNumberOfModules` and `tracedModule` should be `numberOfTracedModules`.